templates/docs/examples/patterns/tables/table-mobile-card.html

```diff
@@ -29,6 +30,23 @@
       <td class="u-align--right"  aria-label="RAM">2 GiB</td>
       <td class="u-align--right"  aria-label="Disks">1</td>
       <td class="u-align--right"  aria-label="Storagelongenoughtocauseellipsis">2TB</td>
+      <td class="has-overflow" aria-label="Actions">
+        <span class="p-contextual-menu--left">
+          <button class="p-contextual-menu__toggle" aria-controls="menu-1" aria-expanded="false" aria-haspopup="true">Actions&hellip;</button>
```

Comment:
It could be useful to have one of the menus expanded by default, so that Percy screenshot can reveal any overflow bugs.